### PR TITLE
Pavel/card table insert component

### PR DESCRIPTION
**Pull request by galloween**
service to pass component as cell data (with ngContent)
*** merge after PR #343

### CODE REVIEW

File: projects/ui-framework/src/lib/layout/card-table/card-table-utils.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/layout/card-table/card-table-utils.ts]: you can remove empty lines

Comment (galloween) [reply projects/ui-framework/src/lib/layout/card-table/card-table-utils.ts]: i dont have them locally, actually. maybe the file got commited before the autoformat did its thing...

**Comment by galloween** [comment]
yes this PR should come after the previous one

**Comment by omrihecht** [comment]
moved to https://github.com/hibobio/bob-style/pull/348

**Pull request closed without merging**